### DISCUSSION TITLE
Mobility and handover implementation

### DISCUSSION THREAD

**Discussion by HaoxinSEU** [srsran/srsRAN_Project#239] [General Help] [answered]
Hi community,

We notice that in the configuration, it's possible to set some mobility related arguments, and an example yaml file `./config/mobility.yaml` is also provided. So we deploy two gNBs using the example configuration file and try to see if the handover between them works. 

When our UE disconnects the old gNB and then connects the new gNB, in AMF we see that it just creates a new gNB-UE connection and removes the old one. We suppose this is not a "real handover" in 5G.

So we are curious that if mobility has already been implemented in srsRAN? If not, do you have any timeline on this feature?

Thanks for your help!

Best regards,
Haoxin

**Comment by ismagom** [comment, marked as answer]
Hi Haoxin, 

We are still developing this feature. It's not fully supported yet.

**Reply by HaoxinSEU** [reply]
Hi Ismael, 

Thanks so much for your reply! BTW, do you have a timeline or schedule when mobility will be available?

Thanks again,
Haoxin

**Reply by connor01M** [reply]
How is this possible in the current framework? How would one do this through zmq?
